ush/config_defaults.yaml

```diff
@@ -1041,6 +1060,12 @@ nco:
   COMIN_BASEDIR: '{{ COMROOT }}/{{ NET }}/{{ model_ver }}'
   COMOUT_BASEDIR: '{{ COMROOT }}/{{ NET }}/{{ model_ver }}'
 
+  #
+  # New additions from RRFS_dev1
+  #
+  NWGES: '{{ OPSROOT }}/nwges'
+  NWGES_BASEDIR: '{{ NWGES }}'
```

Comment:
I still am unclear why there needs to be both an NWGES and NWGES_BASEDIR.

Reply:
That is the way it is in RRFS_dev1. `NWGES_BASEDIR` can be set to `CYCLE_DIR` in community mode and to `NWGES` in NCO mode. 
https://github.com/NOAA-GSL/regional_workflow/blob/feature/RRFS_dev1/ush/setup.sh#L1402-L1413
I am not saying this can not be removed, but that the implementation mirrors it.